README.md

```diff
@@ -81,6 +81,8 @@ This distribution currently supports:
   the [`splunk_hec`
   exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/splunkhecexporter).
 
+The helm chart currently utilizes fluentd by default for Kubernetes logs collection and supports an option to use native OpenTelemetry logs collection for higher throughput and performance. [Go to Logs collection section for more details along with performance benchmarks run internally](#logs-collection)
+
```

Comment:
Suggested change: "The Helm chart currently uses Fluentd by default for Kubernetes logs collection, and supports  an option to use native OpenTelemetry logs collection for higher throughput and performance. See the [logs collection section] for more information, along with performance benchmarks run internally."

For the second sentence, create the hyperlink using only part of the sentence. For example, "[logs collection section]".

Reply:
Thanks Traci, just updated it as part of the latest changes